### PR TITLE
docs: initial commit

### PR DESCRIPTION
**Pull request by StommePoes**

### CODE REVIEW

**Comment by StommePoes** [comment]
@wyseguyonline eyeball: initial doc based heavily on the CONTRIBUTIONS doc in elements.

**Comment by StommePoes** [comment]
Also @loahou04 interested in how you think this reads (not whether you agree with it, lawlz)

File: origami-contributions.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## <a name="commit-message-format"></a> Git Commit Message Format
+
+We let our release changelogs get automatically written based on our Git commit messages by using the [conventional-changelog](https://github.com/ajoslin/conventional-changelog) format to ensure consistency. You are encouraged to use the [commitizen CLI](https://github.com/commitizen/cz-cli) when committing new code, but if you prefer to write your commit messages manually, follow the Commitizen format:
+
+```
+git commit -m "<keyword>(<scope>): <message>"
+```
+
+where the scope is optional, and the keywords are one of the following:
```
Comment (loahou04) [inline origami-contributions.md]: What is scope?

File: origami-contributions.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Contributing to Origami v2 repositories, ElementsSDK, or CompoundsSDK
```
Comment (loahou04) [inline origami-contributions.md]: We should link every readme to this contributions readme as well.  I know that seems redundant...but it ensures that people will see it and not have to go hunt for the "docs" repo

**Comment by StommePoes** [comment]
Hung:
1. I have only the vaguest idea of scope, I use it to give a bit more info about something, like if my commit is a fix and it was a CSS fix, I'd say "fix(css): added semi-colons" or similar. I could make the examples have stuff like that to make it more sensible... the problem I found was trying to find these commitizen rules. Instead almost all the docs I run into are about how to use it in automation, rather than what exactly it is.
2. That is the plan!

**Comment by loahou04** [comment]
If we don't know what it is then why are we using it?

**Comment by StommePoes** [comment]
it's optional! : P

File: origami-contributions.md (hunk 1 above)
Comment (wyseguyonline) [inline origami-contributions.md]: Yes, scope needs to be better explained.

**Comment by StommePoes** [comment]
Well, since I can't find any information about scope, even though it's in the one example I could find, I could leave it out. Then commits that have it will just confuse newbies but since it's not necessary, they don't have to add them in themselves.

**Comment by wyseguyonline** [comment]
Everything I can find seems to point to scope as some secondary way to group changes.  As it is optional, let's skip it for now.  If we need it later, we can better explain what it is and why one might use it.

**Pull request merged**